docker-images/kafka-based/kafka/scripts/kafka_liveness.sh

```diff
@@ -3,7 +3,7 @@ set -e
 
 if [ "$STRIMZI_KRAFT_ENABLED" = "true" ]; then
   # Test KRaft controller process is running
-  . ./kafka_controller_liveness.sh
+  . ./kafka_liveness_kraft.sh
```

Comment:
Since we are now only calling this script from this one place is it worth moving the logic into this file directly, so we don't need to maintain multiple scripts for liveness?

Reply:
+1 to the comment from Kate. Inlining the script might make sense and would make it easier to read and understand what it does.